README.md

```diff
@@ -33,13 +33,19 @@ npm install
 
         npm install
 
-3.  Tag a new version by running the following command: `npm version [ major | minor | patch ]`. Choose `major`, `minor`, or `patch` depending on the kind of update according to [Semantic Versioning](https://semver.org) rules.
+## Tag a Version
```

Comment:
Please turn this back into an enumerated list item. When you made it into a heading, it interrupted the logical flow of the document.